_sass/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,11 @@
+.breadcrumb {
+  margin-bottom: 15px;
+  a {
+    color: darken(#337ab7, 10%);
+    // text-decoration: underline;
```

Comment:
You can remove this since it's not in use. If you need to bring it back you can always look back at the git history

Reply:
Sounds good, I removed it.